### ISSUE TITLE
Any luck passing the provided handle props to cloned children?

### ISSUE THREAD

**Issue by EdmundsEcho** [atlassian/react-beautiful-dnd#2160]
Has anyone had any luck forwarding the `provided` handle props to the child node?

```js
<Draggable>
   ({..}) => <div>{children}</div>
 </Draggable>
 ```

I tried to clone the children (single) to inject the handle props. It did not behave as expected. i.e., I lost the ability to drag altogether :))

Is this the best way to go?

Thanks in advance for any guidance here.

**Comment by danieldelcore** [comment]
Hi there!

Thanks for raising this issue. Can you please create a standalone example on codesandbox.io using our boilerplate 🙏: https://codesandbox.io/s/k260nyxq9v

Without a standalone example, we will not be able to action this one

Cheers!

**Comment by EdmundsEcho** [comment]
My apologies for the delayed response.  Just last week I revised the problem.  With a fresh view, I found another, in hindsight, "obvious" approach.  To move the handle props to the children, I treated children as a function passing it the handle props.  From there, converted the child component to a function that anticipates the handle props.  Simple, pass the props down the chain by converting components to functions (propagating the render-prop pattern).
